code/game/objects/items.dm

```diff
@@ -406,7 +406,7 @@ var/global/objects_thrown_when_explode = FALSE
 			if(!user.put_in_hand_check(src, user.get_active_hand()))
 				return
 		//canremove==0 means that object may not be removed. You can still wear it. This only applies to clothing. /N
-		if(!canremove)
+		if(!canremove && user.is_wearing_item(src))
```

Comment:
Does this affect items that can't be dropped (i.e. handcuffed briefcases)? Doesn't seem like it since user.put_in_hand_check handles it (I think), but good to make sure.

Reply:
`cant_drop` is its own thing that's distinct from `canremove`
What are the controls for handcuffing a briefcase to someone by the way? I haven't done it in game but I can test it.
varediting something to `cant_remove = 1`, and trying to switch which hand its in still doesn't work (as intended)

Reply:
Wonder why canremove is a item var and not a clothing var then. Odd.

Reply:
You can wear some non-clothing items like lolipop sticks and novaflowers.